.github/workflows/test-digital-ocean-1-click-install.yaml

```diff
@@ -0,0 +1,113 @@
+#
+# This is an e2e test to deploy PostHog on DigitalOcean using DigitalOcean's 1-click app install.
+#
+# TODO:
+# - run k8s spec test
+# - run action only when necessary
+# - test the "Securing your 1-click install" https://posthog.com/docs/self-host/deploy/digital-ocean#securing-your-1-click-install
+#
+name: e2e - DigitalOcean "1-click" (install)
+
+#
+# Unfortunately we can test this scenario only after merging to 'main' as we don't have
+# control over the branch used by the DigitalOcean's marketplace 1-click install.
+#
+# ref: https://github.com/digitalocean/marketplace-kubernetes/blob/master/stacks/posthog/deploy.sh
+#
+on:
+  push:
+    # branches:
+    #   - main
```

Comment:
I've left this commented to test this action in this PR. I'm going to uncomment it once I have your ✅  and I see the test passing.